Packs/XMCO/Integrations/Serenety/Serenety_description.md

```diff
@@ -0,0 +1,16 @@
+## XMCO Serenety
+
+To configure an instance of XMCO Serenety integration in Cortex XSOAR:
+
+1. Select `XMCO Serenety Alert` in the Incident type section.
+2. Select `XMCO - Serenety Mapper` in the Mapper section.
+3. Provide 'Server URL'. The default server URL should be sufficient.
+
+4. Provide 'API Key':
+
+   Retrieve your authentication token via the [XMCO LePortail](https://leportail.xmco.fr)
```

Comment:
Does the key require specific permissions?

Reply:
It's require to have an account on our platform but i have presumed that only our clients will install our integration from your marketplace but i could be wrong

Reply:
I meant if the generated API key from your portal may require certain special permission to run this integration. If not, you may resolve this comment. If yes, I suggest to mention that in the description :)